dissasembler.X68

```diff
@@ -98,29 +96,26 @@ OP_CODES:
 OP_START:
         * write address and a tab to str buffer
         MOVE.L  A5,D1   * load address from A5 into D1
-        MOVEQ   #32,D5   * set hex length to long for address printing
```

Comment:
Don't know if removing this is intentional but with the new write ascii method needs to know what size to print 32 for long anything else for word